python_modules/libraries/dagster-pagerduty/dagster_pagerduty/resources.py

```diff
@@ -0,0 +1,25 @@
+import pypd
+
+from dagster import Dict, Field, ResourceDefinition, String
+
+
+class PagerDutyService:
+    def __init__(self, token):
+        self.token = token
+
+    def create_incident(self, summary):
+        pypd.EventV2.create(
+            data={
+                'routing_key': self.token,
+                'event_action': 'trigger',
+                'payload': {'summary': summary, 'severity': 'error', 'source': 'Dagster'},
+            }
+        )
+
+
+def define_pagerduty_resource():
```

Comment:
for these off-the-shelf resources, should we be encouraging a naming convention where it is just 

```define_pagerduty_resource``

or

```pagerduty_resource```

omitting the definition feels slightly more "literate programming" to me but open to discuss

Reply:
fixed